src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  let discount = 0;
+
+  switch (true) {
+    case days >= 3 && days < 7:
+      discount = 20;
+      break;
+    case days >= 7:
+      discount = 50;
+      break;
+  }
+
+  const TOTAL_COST = days * RENT_COST - discount;
+
+  return TOTAL_COST;
```

Comment:
The function name 'calculateRentalCost' is descriptive enough, but the use of a `switch` statement here is not necessary and overcomplicates the logic. A series of `if` statements would be more appropriate, especially since you're checking boolean conditions rather than specific case values.